api/AudioListener.json

```diff
@@ -100,15 +105,15 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioListener/forwardX",
           "support": {
             "webview_android": {
-              "version_added": false,
+              "version_added": "52",
               "notes": "Supports a deprecated way of setting orientation — the <code>setOrientation()</code> method."
```

Comment:
Since these notes have been kept, I'm assuming this means that both this property and the setOrientation() method are still supported (I think this is the case with Fx too, as well). In which case, these notes should start with "Also supports..."?

Or better, just get rid of all these notes and just keep both bits of compat data for the properties and the obsolete method(s)?

Also, Looking at the full file, you've still got the obsolete methods set as "standard_track: true" — this should e false given that they've been removed from the standards.

Reply:
The current editor's draft doesn't list that as deprecated. I'm reaching out to the spec authors.